Tests/PackageGraphTests/DependencyResolverTests.swift

```diff
@@ -827,10 +827,10 @@ private func checkResolution(_ resolver: MockDependencyResolver, constraints: [M
 
     // Check the solution against our oracle.
     if let solution = solution {
-        if maximalSolutions.count != 1 {
+        guard let onlySolution = maximalSolutions.only else {
             return XCTFail("solver unexpectedly found: \(solution) when there are no viable solutions")
         }
-        if solution != maximalSolutions[0] {
+        if solution != onlySolution {
             return XCTFail("solver result: \(solution.map{ ($0.0.identifier, $0.1) }) does not match expected result: \(maximalSolutions[0].map{ ($0.0.identifier, $0.1) })")
```

Comment:
Nice catch! Missed it because it was past my 120 characters viewing limit :) We need to go through the test files and hard-wrap past 120 characters.